source/bibdb/terms.ttl

```diff
@@ -2,8 +2,105 @@ prefix owl: <http://www.w3.org/2002/07/owl#>
 prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 prefix skos: <http://www.w3.org/2004/02/skos/core#>
 prefix kbv: <https://id.kb.se/vocab/>
+prefix idkbse_bibdb: <https://id.kb.se/ns/bibdb/>
 prefix : <https://id.kb.se/term/bibdb/>
 
 :Registrant a skos:Concept ;
     skos:prefLabel "Registrant"@en, "Registrerande"@sv ;
-    rdfs:comment "Bibliotek i Librissamarbetet som skapar och underhåller metadata i den gemensamma datamängden."@sv .
\ No newline at end of file
+    rdfs:comment "Bibliotek i Librissamarbetet som skapar och underhåller metadata i den gemensamma datamängden."@sv .
+
+
+# https://git.kb.se/libris/bibdb/-/blob/develop/libdb/models.py#L262
+:natbib a idkbse_bibdb:LibraryType ;
```

Comment:
One thing that came to mind: What's the convention for enums, should they be capitalized or not?

Reply:
I think we are mixing a bit :slightly_smiling_face:

Reply:
I just tried to keep it simple and use the exact string from BIBDB so that we can match them directly :thinking:

Reply:
That's what I thought!